orchagent/aclorch.cpp

```diff
@@ -3304,6 +3314,21 @@ void AclOrch::doAclRuleTask(Consumer &consumer)
                     break;
                 }
             }
+            // If acl rule is to match TCP_FLAGS, and IP_PROTOCOL is not set
+            // we set IP_PROTOCOL to 6 to match TCP explicitly
+            if (bHasTCPFlag && !bHasIPProtocol)
+            {
+                string attr_name = MATCH_IP_PROTOCOL;
```

Comment:
for v6 traffic, it is called next_header

Reply:
Thanks. Updated.

Reply:
Thanks. Updated.